src/utils/customRegistry.ts

```diff
@@ -0,0 +1,17 @@
+import process from 'process';
+import { LogFunction } from './logger';
+
+let hasLoggedCustomRegistry = false;
+
+export function getCustomRegistry(log: LogFunction = () => {}) {
+  if (process.env.FPL_REGISTRY) {
+    if (!hasLoggedCustomRegistry) {
+      hasLoggedCustomRegistry = true;
```

Comment:
I wonder if instead of using the flag and possibly have in the race condition, could the info message just be logged after [line 216](https://github.com/standardhealth/fhir-package-loader/pull/10/files#diff-51cd61f2399d311debdd2c813260b4a799b0cd6155b8eaa0f1498aebdc5ca281R216) before the `packageUrl` with the custom registry is set?

Reply:
Node.js is single threaded and there aren't even any async calls between checking and setting the variable. I'm not sure we need to worry about this.

Reply:
If the message were logged there, it would get logged each time `mergeDependency` is called. The purpose of this implementation is that it only logs the message once, no matter how many times `mergeDependency` is called.

Reply:
Ohh right. I lost track of where that function was being called from. Thanks!